website/blog/2022-05-26-aluminium.md

```diff
@@ -0,0 +1,246 @@
+---
+author: Rikito Taniguchi
+title: Metals v0.11.6 - Aluminium
+authorURL: https://twitter.com/tanishiking25
+authorImageURL: https://github.com/tanishiking.png
+---
+
+We're happy to announce the release of Metals v0.11.6, which 
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">119</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">46</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+
+## Reduce file watcher memory usage
+Previously, Metals uses `FileTreeRepository` of [swoval](https://github.com/swoval/swoval) for a file watcher, that maintains an in memory cache of some subset of the file system.
+Now, Metals uses `PathWatcher` of [swoval](https://github.com/swoval/swoval) instead. This implementation no longer caches the watched file trees, and Metals consumes smaller amount of memory.
+
+For more details, see the original PR: [\#3758](https://github.com/scalameta/metals/pull/3758)
+
+## (Scala3) Show scaladoc on hover for Scala 3 project
+Now, Metals shows the scaladoc on hover for Scala3 project. (Before this release, Metals unable to show the scaladoc for the symbols from third-party modules).
+
+![hover-scala3](https://i.imgur.com/Svzq5DD.png)
+
+### (Scala3) Support `completionItem/resolve`
+Now, in Scala 3 we get scaladoc and shows default values when moving accross completions. 
+
+![completion-item-resolve](https://i.imgur.com/Tz6AOsx.gif)
+
+## Show parent scaladoc if implementation is returning empty
+
+## Show scaladocs for signature help
+
```

Comment:
@tgodzik Could you fill these sections? Or if you don't feel it doesn't need to write in a release note, feel free to drop :) 

- Show parent scaladoc if implementation is returning empty
- Show scaladocs for signature help

Reply:
```
## Show parent scaladoc if implementation is returning empty

Scaladocs can be inspected whenever you hover, use a completion or signature help. Up until recently we only showed you the documentation if the exact method you are using had the scaladocs written, which meant that if you overrode a method and didn't add the scaladoc comments again we would not show you any documentation. One example of such method could `headOption` on a `List`.

 From this release we will also search the parent method in case the current method's scaladoc are empty. 

## [Scala 3] Show scaladocs for signature help

As mentioned in the previous paragraph, Metals can show you documentation in three different places. That, however, was true only for Scala 2 previously. In this release, we will now show you proper documentation whenever invoking signature help.

As a remainder, signature help is used to indicate what parameters can be used in a method. It should pop up automatically after writing `(`, but you can also invoke it manually. In VS Code that takes the form of `editor.action.triggerParameterHints` command, which can also be bound to a shortcut and by default is.
```

Reply:
https://github.com/scalameta/metals/pull/3957/commits/719ca155165667a15a1dca92fb004eed5885e196 👍